src/tests/strfmt.rs

```diff
@@ -191,6 +191,13 @@ fn test_ignore_missing() {
     run_tests(&values, &vars, &strfmt_ignore);
 }
 
+#[test]
+fn test_trailing_comma() {
```

Comment:
just to be absolutely sure, please also assert the output is as-expected (it equals bar)